engines/standard_tasks/spec/factories/supporting_information_task_factory.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :supporting_information_task, class: 'SupportingInformation::SupportingInformationTask' do
+    phase
+    title "Supporting Information"
+    role "author"
+  end
+end
```

Comment:
We add factories for each card, so that we can show them in the styleguide. See [`populate_styleguide.rb`](https://github.com/Tahi-project/tahi/pull/791/files#diff-1aab98fabd807b5e8a23a63c99580091R21) where these are used.